IFC4x3/Properties/w/WindDirectionRange_1DH_IenR1EWgR3G1pApMRJ/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the range of wind direction the sensor can monitor.
```

Comment:
Since the datatype is an angle, I would propose WindAngleRange and
```suggestion
Indicates the wind angle range the sensor can monitor.
```

Reply:
Resolved: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9